### PR TITLE
feat(config.sls): allow to not manage zone file

### PR DESCRIPTION
**Pull request by sylvainfaivre**
### PR progress checklist (to be filled in by reviewers)

- [ ] Changes to documentation are appropriate (or tick if not required)
- [ ] Changes to tests are appropriate (or tick if not required)
- [ ] Reviews completed

---

#### Primary type

- [ ] `[build]`    Changes related to the build system
- [ ] `[chore]`    Changes to the build process or auxiliary tools and libraries such as documentation generation
- [ ] `[ci]`       Changes to the continuous integration configuration
- [x] `[feat]`     A new feature
- [ ] `[fix]`      A bug fix
- [ ] `[perf]`     A code change that improves performance
- [ ] `[refactor]` A code change that neither fixes a bug nor adds a feature
- [ ] `[revert]`   A change used to revert a previous commit
- [ ] `[style]`    Changes that do not affect the meaning of the code (white-space, formatting, missing semi-colons, etc.)

#### Secondary type

- [x] `[docs]`     Documentation changes
- [ ] `[test]`     Adding missing or correcting existing tests

### Does this PR introduce a `BREAKING CHANGE`?

No.

### Related issues and/or pull requests

https://github.com/saltstack-formulas/bind-formula/issues/114

### Describe the changes you're proposing

Allow to not manage the zone file for a zone listed in configured_zones

### Documentation checklist

- [ ] Updated the `README` (e.g. `Available states`).
- [ ] Updated `pillar.example`.

### Testing checklist

- [ ] Included in Kitchen (i.e. under `state_top`).
- [ ] Covered by new/existing tests (e.g. InSpec, Serverspec, etc.).
- [ ] Updated the relevant test pillar.

### CODE REVIEW

File: pillar.example [hunk 1]
```diff
@@ -173,6 +173,7 @@ bind:
       type: master                                   # Yo don't have define zone again in available_zones.
                                                      # This feature is backward compatibile and only available in debian
       notify: false                                  # if type master you need specify notify true/false
+      managed: true                                  # Set this to false if you don't want Salt to manage this zone file
```
Comment (javierbertoli) [inline pillar.example:176]: @sylvainfaivre if you don't mind, can you extend the comment to something like
```
# Set this to false if you don't want Salt to manage this zone file.
# If this parameter is set to true or is not set at all, the zone will be managed through salt
```
or something that makes it clear that NOT setting the parameter equals to `true` and will be managed?

**Review by javierbertoli — CHANGES_REQUESTED** [review]
@sylvainfaivre I suggested a little doc change, to make usage clearer.

Other than that, it looks good to me.

File: pillar.example (hunk 1 above)
Comment (sylvainfaivre) [reply pillar.example:176]: Sure, added !

Thanks for the review.

**Review by javierbertoli — APPROVED** [review]

**Comment by javierbertoli** [comment]
@sylvainfaivre even if rubocop tests fail, as they're not related to this PR.
Thanks!

**Pull request merged**